fec.config.js

```diff
@@ -14,9 +14,7 @@ module.exports = {
    * Add additional webpack plugins
    */
   plugins: [],
-  routes: {
-    '/api/provisioning': { host: `http://localhost:8000` }
-  },
+  routesPath: process.env.ROUTES_PATH && __dirname + process.env.ROUTES_PATH,
```

Comment:
We already have resolve available here, I think we can use it :)
```suggestion
  routesPath: process.env.ROUTES_PATH && resolve(process.env.ROUTES_PATH),
```